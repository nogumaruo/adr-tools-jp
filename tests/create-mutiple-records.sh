adr new First Decision
adr new Second Decision
adr new Third Decision
ls doc/adr
head -9 doc/adr/002-second-decision.md
head -9 doc/adr/003-third-decision.md
