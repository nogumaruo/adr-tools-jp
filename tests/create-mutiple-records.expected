adr new First Decision
doc/adr/001-first-decision.md
adr new Second Decision
doc/adr/002-second-decision.md
adr new Third Decision
doc/adr/003-third-decision.md
ls doc/adr
001-first-decision.md
002-second-decision.md
003-third-decision.md
head -9 doc/adr/002-second-decision.md
# 2. Second Decision

日付: 1992-01-12

決定者: @xxx, @xxx

## ステータス

提案中
head -9 doc/adr/003-third-decision.md
# 3. Third Decision

日付: 1992-01-12

決定者: @xxx, @xxx

## ステータス

提案中
